### PR TITLE
Update agent README citamon/agent-cita-exporter port to 1923

### PR DESCRIPTION
**Pull request by sunfjun**
From agent/cita_exporter/Dockerfile ENV PORT=1923

### CODE REVIEW

**Review by jiangxianliang007 — APPROVED** [review]

**Review by mfuuzy — APPROVED** [review]

File: agent/README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ citamon/agent-cita-exporter
 3、查看数据采集信息
 ```
 #citamon_agent_cita_exporter
-curl http://localhost:1920/metrics/cita
+curl http://localhost:1923/metrics/cita
```
Comment (rainchen) [inline agent/README.md]: this change is not expected, 1920 is the exposed port for agent, see https://github.com/citahub/cita-monitor#ports-default-config

File: agent/README.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Tips：
 ```
 docker run -d --name="citamon_agent_cita_exporter_1337" \
 --pid="host" \
--p 1923:1920 \
+-p 1923:1923 \
```
Comment (rainchen) [inline agent/README.md]: this should be `-p 1920:1920 \`

File: agent/README.md (hunk 1 above)
Comment (rainchen) [reply agent/README.md]: @mfuuzy @jiangxianliang007 why do you agree with this change?

Comment (jiangxianliang007) [reply agent/README.md]: because he changed the document of starting cita_exporter independently, without using docker-compose, without using nginx proxy.

Comment (rainchen) [reply agent/README.md]: I see, this is for `### 单个容器部署`.
But I think we should change `-p 1923:1920 \` to `-p 1920:1923 \` to keep using the same endpoint as "http://localhost:1920/metrics/cita"

File: agent/README.md (hunk 2 above)
Comment (rainchen) [inline agent/README.md]: @sunfjun please change `-p 1923:1920` \ to `-p 1920:1923 \`, and undo the line 95 change

Comment (rainchen) [reply agent/README.md]: and squash your commits into 1 commit

**Review by rainchen — CHANGES_REQUESTED** [review]
please update

File: agent/README.md (hunk 2 above)
Comment (sunfjun) [reply agent/README.md]: Has been updated.

**Review by rainchen — CHANGES_REQUESTED** [review]
please change your PR's base branch to develop, since we're using git-flow.

**Review by rainchen — APPROVED** [review]
thanks for the fix

**Pull request merged**